src/run.py

```diff
@@ -20,6 +20,10 @@ def parse_args():
     parser.add_argument("-w", "--webhook", help="A valid HTTP url for a POST request.", dest="webhook_url")
     parser.add_argument("-i", "--chat-id", help="Telegram ID number for the chat room", dest="chat_id")
 
+    # pushover-related arguments
```

Comment:
I feel as though the arguments might get a bit confusing over time. For example, this PR adds '-t' and '-u' -- which only apply to Pushover. Any thoughts on a better way to structure the arguments?

Reply:
@EricJMarti Any thoughts?